src/index.js

```diff
@@ -1,3 +1,12 @@
+import express from "express";
+import helmet from "helmet";
 import { getDisplayString } from "./render";
 
 console.log(getDisplayString()); // eslint-disable-line no-console
+
+const app = express();
+app.use(helmet());
+
+app.get('/', (req, res) => res.send('Hello World!'));
+
+app.listen(3000, () => console.log('Server is listening on port 3000...')) // eslint-disable-line no-console
```

Comment:
Lets add a proper logger. I would not want eslint comments in the code

Reply:
If you hide the implementation behind some functions it shouldn't matter which one we use